src/block-management/block-manager.js

```diff
@@ -172,21 +188,21 @@ export class BlockManager extends React.Component<PropsType, StateType> {
 
 	renderHeader() {
 		return (
-			<View style={ styles.titleContainer }>
-				<PostTitle
-					setRef={ ( ref ) => {
-						this.titleViewRef = ref;
-					} }
-					title={ this.props.title }
-					onUpdate={ this.props.setTitleAction }
-					placeholder={ __( 'Add title' ) } />
-			</View>
+			<PostTitle
+				onRef={ ( ref ) => {
```

Comment:
Nitpick: Can we name this `ref` instead of `onRef`? Since it seems to be more standard in React.

Reply:
done with 67b509a